input/fsh/patient.fsh

```diff
@@ -46,10 +58,19 @@ Description: "Sara Alert outputs additional extensions on Patient resources"
 * identifier ^slicing.discriminator.path = "system"
 * identifier ^slicing.rules = #open
 * identifier ^slicing.description = "Slice to distinguish the identifier for State/Local ID."
-* identifier contains StateLocalId 0..1 MS
+* identifier contains
+  StateLocalId 0..1 MS and
+  CDCId 0..1 MS and
+  NNDSSId 0..1 MS
 * identifier[StateLocalId].system = "http://saraalert.org/SaraAlert/state-local-id"
 * identifier[StateLocalId] ^short = "The State/Local ID assigned by the state or jurisdiction’s monitoring policies."
 * identifier[StateLocalId] ^comment = "The State/Local ID is the variable to represent the assigned state or local ID of the monitoree. This field is determined from the assigned state or jurisdiction’s monitoring policies."
+* identifier[CDCId].system = "http://saraalert.org/SaraAlert/cdc-id"
+* identifier[CDCId] ^short = "The identifier to represent the Centers for Disease Control and Prevention assigned ID of the monitoree. "
+* identifier[CDCId] ^comment = "CDC ID is the variable to represent the Centers for Disease Control and Prevention assigned ID of the monitoree. This field is determined from CDC’s guidance."
+* identifier[NNDSSId].system = "http://saraalert.org/SaraAlert/nndss-id"
+* identifier[NNDSSId] ^short = "The assigned NNDSS ID or Case ID of the monitoree."
+* identifier[NNDSSId] ^comment = "The NNDSS ID is the variable to represent the assigned NNDSS ID or Case ID of the monitoree. This field is determined from the assigned jurisdiction’s monitoring policies."
```

Comment:
I'd spell out "National Notifiable Diseases Surveillance System (NNDSS)" at least once to introduce the acronym.

Reply:
Done in https://github.com/SaraAlert/saraalert-fhir-ig/pull/14/commits/f84f9a892fd52672226469a8caae47d26a01e2fc.